src/audiomixcontroller/DefaultAudioMixController.ts

```diff
@@ -51,8 +53,19 @@ export default class DefaultAudioMixController implements AudioMixController {
         // @ts-ignore
         const oldSinkId: string = this.audioElement.sinkId;
         if (newSinkId !== oldSinkId) {
-          // @ts-ignore
-          this.audioElement.setSinkId(newSinkId);
+          if (new DefaultBrowserBehavior().hasChromiumWebRTC()) {
```

Comment:
Instead of creating new instance every time the API is used, I suggest to create member variable and initialize it only once.

Reply:
Fixed

Reply:
Done, created an instance variable to use instead